data/coalition/heliarch intro.txt

```diff
@@ -0,0 +1,1845 @@
+				launch
+	npc board
+		personality timid plunders entering target
+		government "Lunarium"
+		ship "Saryd Sojourner (Bombardments)" "Hodoviah"
+		conversation
+			`Unlike the other two vessels, this ship is equipped with Heliarch weaponry. Sadly, you can neither verify the equipment it is using nor check for hostages, as a self destruction sequence starts when you're about to enter it.`
+				launch
+	npc board
+		personality timid plunders entering target
+		government "Lunarium"
+		ship "Arach Spindle (Plasma)" "Aramitess"
+		conversation
+			`This ship is equipped with Plasma Turrets from human space, the very weapons the Free Worlds developed. Unfortunately, it initiates a self destruction sequence as you're about to board it, so you cannot verify if it had any hostages.`
+				launch
+	npc board
+		personality timid plunders entering target
+		government "Lunarium"
+		ship "Kimek Thistle (Torpedoes)" "Resheph"
+		conversation
+			`This ship is equipped with Torpedo Launchers. Somehow, the criminals have acquired human outfits. You're about to enter the ship and look for any hostages that might have been there, but you rush back as a self destruction sequence starts.`
+				launch
+	on complete
+		"reputation: Lunarium" = 1
+		event "resistance detained rb"
+		"assisted heliarch" ++
+		payment 1320000
+		conversation
+			`Though you never managed to find out if there were any hostages on board, the Heliarchs stationed at <planet> still herald you as a hero for "detaining the thugs".`
+			`	They immediately set to repair your ship, and you're still on alert for a few hours in case any more ships come through, but everything appears to be normal. The station returns to normality as well: communications and other functions are restored.`
+			`	As soon as the commlinks are back, the Heliarchs contact Consul Plortup and inform him of what happened. Shortly after, <payment> are added to your account.`
+			`	"Humiliating, it would be, had under such circumstances, the criminal ships escaped. Done well, you have, Captain," he says.`
+			`	He discusses with you the outfits that the ships were using and is equally alarmed that they had not only Heliarch equipment, but had also acquired human weapons.`
+			`	"To the other consuls, this information I will bring. Further tasks, of you we may ask, but for now, done your work is, Captain," he ends the transmission after a short salute.`
+
+event "resistance fleeing rb"
+	system "12 Autumn Above"
+		remove fleet "Heliarch"
+
+event "resistance detained rb"
+	system "12 Autumn Above"
+		add fleet "Heliarch" 800
+
+
+
+mission "Heliarch Drills 1"
+	name "Refuelling Drill"
+	description "Escort the two Heliarch Judicators to <destination>, as part of a drill testing the Coalition's refuelling capabilities in desperate scenarios."
```

Comment:
Suggested this change because the two different phrases mean the same thing, but "emergency situations" fits a little better when talking about the need to refuel instead of things sounding desperate. Purely subjective though. 

```suggestion
	description "Escort the two Heliarch Judicators to <destination>, as part of a drill testing the Coalition's refuelling capabilities in emergency situations."
```